policy/policy.sample.json

```diff
@@ -0,0 +1,78 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
```

Comment:
The AppliedTo is needed if we need policy to be flexible enough to be extended to openstack and mesos, in latter case the notion of Principal/Role leads to complicated tenant/user relationship which our policy should be able to handle.